app/controllers/application_controller.rb

```diff
@@ -1,7 +1,17 @@
-class ApplicationController < ActionController::API
+class ApplicationController < ActionController::Base
   include GDS::SSO::ControllerMethods
 
   before_action :require_signin_permission!
 
-  protect_from_forgery with: :exception
+  rescue_from ActionController::ParameterMissing do |e|
+    render json: { error: { message: e.message } }, status: 400
+  end
```

Comment:
Yeah, I see we can use the `permit` feature, but I would just say we ignore extra parameters that we're given. I'm not passing the `params` hash around to anything directly, so there's no risk of accidental injection, but perhaps this is me fighting Rails.